tasks/run.js

```diff
@@ -129,14 +129,14 @@ function runRouter(opts) {
 		});
 
 	return configureAndSpawn(envVars, function (env) {
-		var bin = opts.https ? 'next-router-https' : 'next-router';
+		var bin = opts.https ? `${opts.router}-https` : `${opts.router}`;
```

Comment:
Why is `${opts.router}` inside back-ticks for the last bit of the statement? Could just refer to `opts.router`.